metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/api/AssemblyXmlRetrieverThroughEntrezApi.java

```diff
@@ -29,10 +29,12 @@
 
     private static final String ENTREZ_API_KEY_QUERY = "&api_key={entrezApiKey}";
 
-    private static String entrezApiIdRetrievalUrl =
+    // URL to obtain an internal Entrez ID from, given a GCF assembly accession.
```

Comment:
I suggested the other comment style not because it was multi-line, but to make it the attribute's Javadoc.

Reply:
ok got it 👍